cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -74,12 +74,7 @@ def upstream(self):
 
     @property
     def sorted_branches(self):
-        def version_from_branch(branch):
-            try:
-                return tuple(map(int, re.match(r'^.*(?P<version>\d+(\.\d+)+).*$', branch).groupdict()['version'].split('.')))
-            except AttributeError as attr_err:
-                raise ValueError(f'Branch {branch} seems to not have a version in its name.') from attr_err
-
+        """Return the branches to cherry-pick to, sorted by version"""
```

Comment:
Nitpick: I prefer PEP 257 compatible docstrings. Please add a period at the end of the sentence.